rom/laghos.cpp

```diff
@@ -1289,34 +1349,57 @@ int main(int argc, char *argv[])
                 if (myid == 0)
                     cout << "ROM online at t " << t << ", dt " << dt << ", romS norm " << romS.Norml2() << endl;
 
+                if (rom_build_database && !std::isfinite(romS.Norml2()))
```

Comment:
What would cause `romS` to not be finite?

Reply:
If MFEM does not converge for some reason, the run will take forever. This kills the run at the step after it does not converge, since romS will be full of NAN's.

Reply:
This makes sense. A comment briefly explaining that would be helpful.